frontend/src/views/AppComponents/AppChatroom.js

```diff
@@ -50,6 +50,20 @@ class AppChatroom extends React.Component {
       console.log(`Received message ${msg}`)
       // append to messages list
       this.setState({ messages: [...this.state.messages, { id: msg.id, sender_uid: msg.sender_uid, text: msg.text }] })
+
+      // if sender is other person, changes favicon to notif
+      if (msg.sender_uid !== this.props.getUserId()) {
+        document.getElementById('tabicon').href = "favicon-notif/favicon.ico";
+      }
+
+      // changes favicon back to original
+      function switchBack() {
+        document.getElementById('tabicon').href = "favicon/favicon.ico";
+       }
+
+      // after clicking anywhere on screen, change favicon back
+      document.onclick = function(){switchBack();};
```

Comment:
This works, but it would be even cooler if we'd automatically switch the favicon back once the tab is opened. Check out the link from [https://stackoverflow.com/questions/49902883/check-if-the-browser-tab-is-in-focus-in-reactjs](StackOverflow) for an example of how to do this.

Reply:
Done